test/test_transformers.py

```diff
@@ -2570,6 +2570,7 @@ def test_mem_eff_backwards_determinism(self, device):
 
     # verified passing successfully on H100
     @unittest.skipIf(not PLATFORM_SUPPORTS_MEM_EFF_ATTENTION, "Does not support SDPA")
+    @unittest.skipIf(IS_JETSON, "causing sigkill on Jetson")
```

Comment:
Do you mind creating an issue to track the fix for those crashes and mention it here?

Reply:
For sigkill it may just be a case of the test requiring too much host memory

Reply:
Yeah the sigkill seems to be a host memory issue; running the test individually is fine but running the full comprehensive parametrize during the CI run leads to the sigkill.